src/index.css

```diff
@@ -1,27 +1,286 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;700&family=Roboto+Slab:wght@300;400&display=swap');
```

Comment:
- [ ] It would be best if you remove the outer white space over the App, kindly modify your styling so the page takes up all the space

|Your design|How it should be|
|-|-|
|![Screenshot_20220716_112924](https://user-images.githubusercontent.com/25227022/179351287-f92fbf1b-cd00-4b74-8b12-4815eb6c0b97.png)|![Screenshot_20220716_112949](https://user-images.githubusercontent.com/25227022/179351288-8769f63e-4af6-48e5-9360-5594ba81fe12.png)|

Reply:
Hi @Ismailco, thanks for the suggestion, but I don't have that white space when I check my app here.

Reply:
![Capture](https://user-images.githubusercontent.com/78251927/179352910-90b4d82a-6f45-47ab-bad6-27b36ab71f08.PNG)

Reply:
Your page should be like this: ![](https://user-images.githubusercontent.com/25227022/179351288-8769f63e-4af6-48e5-9360-5594ba81fe12.png)